From: Sergey Vojtovich <svoj@mariadb.org>
Desciption: Make auth_socket built-in by default
 Remove this the day passwordless root account becomes the default in
 upstream too and not just in Debian packaging.
Date: Fri, 18 Dec 2015 16:31:05 +0400
Subject: [PATCH] Fixed auth_socket static compilation
Origin: https://github.com/MariaDB/server/commit/3402f7a85364e26fe85dfff3b5d803a8497042aa.patch
Bug: https://jira.mariadb.org/browse/MDEV-8375

--- a/mysql-test/mysql-test-run.pl
+++ b/mysql-test/mysql-test-run.pl
@@ -3644,6 +3644,9 @@ sub mysql_install_db {
     mtr_tofile($bootstrap_sql_file,
          "DELETE FROM mysql.user where user= '';\n");
 
+   mtr_tofile($bootstrap_sql_file,
+             "UPDATE mysql.user SET plugin='';\n");
+
     # Create mtr database
     mtr_tofile($bootstrap_sql_file,
          "CREATE DATABASE mtr CHARSET=latin1;\n");
